src/gsi/netcdfgfs_io.f90

```diff
@@ -2256,6 +2261,447 @@ subroutine write_atm_ (grd,sp_a,filename,mype_out,gfs_bundle,ibin)
 
   end subroutine write_atm_
 
+  subroutine write_atm_pert_ (grd,sp_a,filename,mype_out,gfs_bundle,ibin)
```

Comment:
The code in this subroutine here references to `write_atm` or `write_ncatm`.
The `print` statements etc would be cleaner if it used `my_name` which should also be corrected to `WRITE_GFSNCATM_PERT`.

The code here works.  These changes will likely improve readability and help debugging.

Reply:
my_name changed to WRITE_GFSNCATM_PERT